docs/pages/x/api/tree-view/tree-item.json

```
@@ -40,6 +38,12 @@
       "name": "icon",
       "description": "The icon to display next to the tree node's label.",
       "class": null
+    },
+    {
+      "name": "groupTransition",
+      "description": "The component that renders the children of the item.",
```

this seems like it is wrong ...

---

Oh sry ... I think it should mention that it is meant for adding a transition, or am I missing something? Before it was the `group`, where the current description made sense, but with the new name it does not anymore. I know that it is still used to render the children though

---

How about the following?

```suggestion
      "description": "The component that animates to appearance / disappearance of the item's children.",
```